code/game/objects/structures/inflatable.dm

```diff
@@ -495,6 +498,22 @@
 		if(do_after_many(user,list(target,src),20)) //Twice the normal time
 			enter_shelter(target)
 
+
+/obj/structure/inflatable/shelter/MouseDropFrom(over_object, src_location, var/turf/over_location, src_control, over_control, params)
```

Comment:
Ditto. Also why do you only have it on one of the args?

Reply:
dirty copypasta from mecha.dm inbetween two rounds

Reply:
I removed it from this proc so it doesn't stick out like a sore thumb. Is that good enough?